augur/types.py

```diff
@@ -1,8 +1,27 @@
 import enum
 
 
+class ArgparseEnum(enum.Enum):
+    """
+    Intended to be used as a parent class for any enum representation of
+    string values to be used with argparse options.
+
+    Can be replaced by :py:class:`enum.StrEnum` once Augur's minimum supported
+    Python version is 3.11.
```

Comment:
Cool! I learned more about enums from this PR! :D

Reply:
😄